app/scripts/extract_nyc_data.py

```diff
@@ -0,0 +1,199 @@
+import os
+from typing import List, Dict, Tuple
+from requests.models import Response
+from collections.abc import Callable
+import requests
+import time
+import datetime
+import pytz
+from dotenv import load_dotenv
+from app.route_rangers_api.models import (
+    TransitRoute,
+    RidershipRoute,
+    TransitStation,
+    RidershipStation,
+)
+
+#########################
+# Load and define variables
+#########################
+load_dotenv()
+
+DATA_PORTAL_APP_TOKEN = os.getenv("DATA_PORTAL_APP_TOKEN")
+TEST_DATA_DIR = os.getenv("TEST_DATA_DIR")
+
+REQUEST_DELAY = 0.2
+RESULTS_PER_PAGE = 50000  # Max number of results for API
+TIMEOUT = 30
+
+COLS_TO_KEEP = {"BUS_RIDERSHIP": [], "SUBWAY_RIDERSHIP": []}
+
+DATASETS = {
+    "BUS_RIDERSHIP": {
+        "URL": "https://data.ny.gov/resource/kv7t-n8in.json?",
+        "GROUP_BY": "date, bus_route",
+        "OBS_LEVEL": "bus_route",
+    },
+    "SUBWAY_RIDERSHIP": {
+        "URL": "https://data.ny.gov/resource/wujg-7c2s.json?",
+        "COLS_TO_KEEP": ["date", "station_complex_id", "ridership", "transfers"],
+        "GROUP_BY": "date, station_complex_id",
+        "OBS_LEVEL": "station_complex_id",
+    },
+}
+
```

Comment:
do these come in as UTC and if so do you want to keep them that way since that is how the db is expecting?